src/runtime/manifest.md

```diff
@@ -174,35 +173,37 @@ Can include how particle inputs are connected to handles:
 ```
 ...
 recipe
-  map as handle1                 // maps a store external to the arc; changes in the
+  handle1: map                   // maps a store external to the arc; changes in the
                                  // external store will be reflected locally, but the
                                  // local handle is read-only.
-  use as handle2                 // uses some handle already in the Arc
-  create as handle3              // creates a new empty handle
-  copy ImportedHandle as handle4 // creates a handle on a new store populated with entries
+  handle2: use                   // uses some handle already in the Arc
+  handle3: create                // creates a new empty handle
+  handle4: copy ImportedHandle   // creates a handle on a new store populated with entries
                                  // from a store defined in or imported by the manifest
                                  
   SomeParticle
-    param1 <- handle1  // bind's SomeParticle's 'in' param1 connection to handle1
-    param2 -> handle2  // binds 'out'
-    param3 = handle3   // binds 'inout'
+    param1: reads handle1   // bind's SomeParticle's input param1 connection to handle1
+    param2: writes handle2  // binds output connection
+    param3: reads writes handle3   // binds input and output connection
```

Comment:
This can be `any` if we wish to show the equivalent of the '=' behavior.

Reply:
updated, thanks!